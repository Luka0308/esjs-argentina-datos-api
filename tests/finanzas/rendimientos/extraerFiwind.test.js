import { describe, expect, it } from 'vitest'
import { extraerFiwind } from '@/finanzas/rendimientos/extraerFiwind.esjs'
// import { guardarRendimientos } from '@/finanzas/rendimientos/guardarRendimientos.esjs'

describe('extraerFiwind', () => {
  it('extrae los rendimientos', async () => {
    try {
      const items = await extraerFiwind()

      expect(items).toMatchSnapshot()
    } catch (error) {
      console.error(error)
    }
  })
})
