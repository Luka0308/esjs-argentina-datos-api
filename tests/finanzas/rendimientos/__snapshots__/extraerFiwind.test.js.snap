// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`extraerFiwind > extrae los rendimientos 1`] = `
[
  {
    "apy": 0.5,
    "fecha": "2022-01-25",
    "moneda": "BTC",
  },
  {
    "apy": 0.5,
    "fecha": "2022-01-25",
    "moneda": "ETH",
  },
  {
    "apy": 2.5,
    "fecha": "2022-01-25",
    "moneda": "USDT",
  },
  {
    "apy": 2.5,
    "fecha": "2022-08-22",
    "moneda": "USDC",
  },
  {
    "apy": 2.5,
    "fecha": "2022-01-25",
    "moneda": "DAI",
  },
  {
    "apy": 1,
    "fecha": "2022-04-18",
    "moneda": "ADA",
  },
  {
    "apy": 2,
    "fecha": "2022-07-18",
    "moneda": "DOT",
  },
  {
    "apy": 2,
    "fecha": "2022-08-26",
    "moneda": "MATIC",
  },
]
`;
